src/engine/ruleset/integrations/docker-listener/README.md

```diff
@@ -0,0 +1,51 @@
+# Wazuh Docker Listener Integration
```

Comment:
Please complete the documentation with more detailed information about the wodle and the service covered:
- References to the service and wodle
- Configuration details